src/chapters/booleans-and-conditionals/equality.rst

```diff
@@ -0,0 +1,118 @@
+.. _equality:
+
+Equality
+========
+
+Loose Equality With ``==``
+--------------------------
+
+.. index:: equality, ==
+
+In the section :ref:`booleans`, we learned about the comparison operators ``==`` and ``!=``, which test whether two values are equal or not equal, respectively. We also tipped you off to some quirks of the ``==`` operator. Specifically, when its operands are of different types, then the returned boolean value may not be what you expect.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(7 == "7");
+      console.log(0 == false);
+      console.log(0 == '');
+
+   **Output**
+
+   ::
+
+      true
+      true
+      true
+
+.. index::
+   single: equality; loose
+
+When we give ``==`` operands of different data types, it will still try to make a comparison. In order to properly do so, the values must be both be of the same type. JavaScript will implicitely convert the operands so that they are of the same type, and then compare the converted values to generate ``true`` or ``false``. For this reason, the ``==`` operator is often said to measure **loose equality**.
```

Comment:
Second sentence suggestion: "To do this, JavaScript will implicitly convert the operands into the *same data type* before making the comparison and returning ``true`` or ``false``."

Reply:
Yeah, that's better. Fixed.